source/module.c

```diff
@@ -29,6 +29,53 @@
 
 #include <memoryobject.h>
 
+static struct aws_logger s_logger;
+static bool s_logger_init = false;
+
+PyObject *aws_py_io_init_logging(PyObject *self, PyObject *args) {
+    (void)self;
+
+    if (s_logger_init) {
+        Py_RETURN_NONE;
+    }
+
+    s_logger_init = true;
+
+    struct aws_allocator *allocator = aws_crt_python_get_allocator();
+
+    int log_level = 0;
+    const char *file_path = NULL;
+    Py_ssize_t file_path_len = 0;
+
+    if (!PyArg_ParseTuple(args, "bs#", &log_level, &file_path, &file_path_len)) {
+        PyErr_SetNone(PyExc_ValueError);
+        return NULL;
+    }
+
+    struct aws_logger_standard_options log_options = {
+        .level = log_level,
+        .file = NULL,
+        .filename = NULL,
+    };
+
+    Py_ssize_t stdout_len = (Py_ssize_t)strlen("stdout");
+
+    Py_ssize_t cmp_len = file_path_len > stdout_len ? stdout_len : file_path_len;
+
+    if (!memcmp("stdout", file_path, (size_t)cmp_len)) {
```

Comment:
Should we allow these to be case-insensitive?

Reply:
I don't think the extra branches are worth it, and it's written in lowercase in python anyways.